app/analytics/query.ts

```diff
@@ -41,6 +43,10 @@ function accumulateCountsFromRowResult(
     if (row.isVisitor == 1) {
         counts.visitors += Number(row.count);
     }
+    if (row.bounce && row.bounce != 0) {
```

Comment:
I think to reduce complexity, I would make this if statement look exactly like the others.

Do:

```if (row.bounce == 1 || row.bounce == -1) {```

It also lets you remove your comment (`// bounce is either 1 or -1`), because it's very clear there in the code.